.github/workflows/performance.yml

```diff
@@ -0,0 +1,160 @@
+name: Performance Tests
+
+on:
+  push:
+    branches:
+      - trunk
+      - '6.[1-9]'
+      - '[7-9].[0-9]'
+    tags:
+      - '[6-9]+.[0-9]'
+      - '[0-9]+.[0-9].[0-9]+'
+  pull_request:
+    branches:
+      - trunk
+      - '6.[1-9]'
+      - '[7-9].[0-9]'
+
+# Cancels all previous workflow runs for pull requests that have not completed.
+concurrency:
+  # The concurrency group contains the workflow name and the branch name for pull requests
+  # or the commit hash for any other events.
+  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+  cancel-in-progress: true
+
+env:
+  LOCAL_DIR: build
+
+jobs:
+  # Runs the performance test suite.
+  #
+  # Performs the following steps:
+  # - Sets environment variables.
+  # - Checks out the repository.
+  # - Sets up Node.js.
+  # - Log debug information.
+  # - Installs npm dependencies.
+  # - Builds WordPress to run from the `build` directory.
+  # - Starts the WordPress Docker container.
+  # - Logs the running Docker containers.
+  # - Logs Docker debug info.
+  # - Installs WordPress.
+  # - Installs the Server Timing MU plugin.
+  # - Install wordpress importer plugin and import dummy data.
```

Comment:
After we're happy with any changes below we should do a full step through of these docs just to confirm they're still accurate.

Reply:
I've updated this list to match the names of each step for consistency in f5446a5